Bottom: 5413e6387efb561a6d35b785465f2bcd5d154b0e
Top:    fe1d01c353992b61fcdc1a0ab729805c12e9592f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 16:22:25 +0000

Refresh of l-add-define-yy_no_input-to-av

---

diff --git a/src/gts/pgraph.c b/src/gts/pgraph.c
index 2c13c1e..6d1f619 100644
--- a/src/gts/pgraph.c
+++ b/src/gts/pgraph.c
@@ -30,8 +30,8 @@ static void gnode_split_destroy (GtsObject * object)
     gts_object_destroy (GTS_OBJECT (ns->n));
   }
   else {
-    GtsGNode * n1 = GTS_GNODE_SPLIT_N1 (ns);
-    GtsGNode * n2 = GTS_GNODE_SPLIT_N2 (ns);
+    /* GtsGNode * n1 = GTS_GNODE_SPLIT_N1 (ns); */
+    /* GtsGNode * n2 = GTS_GNODE_SPLIT_N2 (ns); */
 
     g_warning ("Memory deallocation for GtsGNodeSplit not fully implemented yet: memory leak!");
   }
diff --git a/src/parse_l.l b/src/parse_l.l
index b1b67a8..c9d96c8 100644
--- a/src/parse_l.l
+++ b/src/parse_l.l
@@ -62,6 +62,8 @@ RCSID("$Id$");
 #include "parse_l.h"
 #include "parse_y.h"
 
+#define YY_NO_INPUT
+
 /* ---------------------------------------------------------------------------
  * some shared parser identifiers
  */
